typescript/src/pre0/McapPre0Reader.test.ts

```diff
@@ -222,10 +222,10 @@ describe("McapReader", () => {
 
         ...record(RecordType.CHUNK, [
           ...uint64LE(1n), // decompressed size
-          ...uint32LE(crc32(new Uint8Array([RecordType.CHANNEL_INFO]))), // decompressed crc32
+          ...uint32LE(crc32(new Uint8Array([RecordType.CHANNEL]))), // decompressed crc32
```

Comment:
I would rather we avoid any changes in the `pre0` folder as it's effectively frozen now.